js/models/messages.js

```diff
@@ -1125,38 +1111,43 @@
 
       const quoteWithData = await loadQuoteData(this.get('quote'));
       const previewWithData = await loadPreviewData(this.get('preview'));
+      const chatMessage = new libsession.Messages.Outgoing.ChatMessage({
+        body,
+        timestamp: this.get('sent_at'),
+        attachments,
```

Comment:
Same as the similar comment above saying attachments are not in the correct format here